crypto/fipsmodule/cipher/aead.c

```diff
@@ -285,3 +285,20 @@ int EVP_AEAD_CTX_tag_len(const EVP_AEAD_CTX *ctx, size_t *out_tag_len,
   *out_tag_len = extra_in_len + ctx->tag_len;
   return 1;
 }
+
+// EVP_AEAD_iv_from_ipv4_nanosecs computes a deterministic IV compliant with
+// NIST SP 800-38D, built from an IPv4 address and the number of nanoseconds
+// since boot, writing it to |out_iv|. It returns one on success or zero for
+// error.
+int EVP_AEAD_iv_from_ipv4_nanosecs(const uint32_t ipv4_address,
```

Comment:
slight naming nit, but 
```suggestion
int EVP_AEAD_get_iv_from_ipv4_nanosecs(const uint32_t ipv4_address,
```
would be nice, and aligns with the naming of the `EVP_AEAD_CTX_get_iv` function above (Line 262).